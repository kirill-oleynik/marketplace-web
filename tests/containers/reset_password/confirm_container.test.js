const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const ResetPasswordConfirmContainer = require('../../../app/containers/reset_password/confirm_container').default;

const fakeStore = (state = {}) => ({
  subscribe() {},
  dispatch() {},
  getState() { return state; }
});

const resetPassword = {
  errors: {}
}

describe('#render', () => {
  test('it renders correctly', () => {
    const store = fakeStore({
      resetPassword
    });

    const component = shallow(
      <ResetPasswordConfirmContainer
        t={(translate) => translate}
        resetPasswordConfirm={() => {}}
        handleFormCancel={() => {}}
      />,
      { context: { store } }
    )

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
