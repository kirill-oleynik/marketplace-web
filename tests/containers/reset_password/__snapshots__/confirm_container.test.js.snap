// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<Translate(ResetPasswordConfirmContainer)
  handleFormCancel={[Function]}
  resetPasswordConfirm={[Function]}
  t={[Function]}
/>
`;
